e2e/js/js-cdn/project.json

```diff
@@ -0,0 +1,95 @@
+{
+    "name": "js-cdn",
```

Comment:
if you set the implicitDependencies properly in here then you can have this run automatically IFF JS SDK is changed

Reply:
gotcha, sounds good